src/publish/s3.js

```diff
@@ -3,9 +3,18 @@ const winston = require("winston-color")
 const zlib = require("zlib")
 const config = require("../config")
 
-const S3 = new AWS.S3()
+// This is the case where using custom s3 api-like services like minio.
+let conf = {
```

Comment:
This is a nit, but could we use the `const` keyword since this object doesn't get modified anywhere?